local ReplicatedStorage = game:GetService("ReplicatedStorage")
local Iris = require(ReplicatedStorage.Packages.iris)


return function ()
    Iris.Window({" Properties : "}) 
        Iris.Tree({" Metadata "})
           Iris.InputText({" Name "})
        Iris.End()
        Iris.Separator()
        Iris.Tree({" Physics "})
           Iris.InputVector3({" Position "})
           Iris.InputVector3({" Scale "})
           Iris.Checkbox({" Anchored "})
           Iris.Checkbox({" CanCollide "})
        Iris.End()
        Iris.Separator()
        Iris.Tree({" Rendering "})
            Iris.InputColor3({" Color "})
            Iris.SliderNum({" Transparency ", 1})
        Iris.End()
        Iris.Separator()
        Iris.Tree({" Components "})
           Iris.Button({" Add Texture "})
           Iris.Button({" Add Image "})
        Iris.End()
    Iris.End()
end